### PR TITLE
Small updates for 1.13

### PR DESCRIPTION
**Pull request by ProZeratul**

### CODE REVIEW

File: src/ck3/data/messages.rs [hunk 1]
```diff
@@ -62,8 +62,32 @@ impl DbKind for MessageFilterType {
         let loca = format!("message_filter_{key}");
         data.verify_exists_implied(Item::Localization, &loca, key);
 
-        vd.field_choice("display", &["feed", "toast", "hidden"]);
+        vd.field_choice("display", &["feed", "toast", "hidden", "popup"]);
         vd.field_bool("always_show");
         vd.field_bool("auto_pause");
+        vd.field_integer("sort_order");
+        vd.field_item("group", Item::MessageFilterGroupType);
```
Comment (amtep) [inline src/ck3/data/messages.rs:69]: Where did you get these two fields? I don't see them in the docs or in the game scripts

File: src/ck3/data/messages.rs [hunk 2]
```diff
@@ -62,8 +62,32 @@ impl DbKind for MessageFilterType {
...
+    }
+}
+
+
+#[derive(Clone, Debug)]
+pub struct MessageFilterGroupType {}
+
+inventory::submit! {
+    ItemLoader::Normal(GameFlags::Ck3, Item::MessageFilterGroupType, MessageFilterGroupType::add)
```
Comment (amtep) [inline src/ck3/data/messages.rs:78]: When you add a new `Item` type, you have to add it in `src/items.rs` as well. In two places: once in the `Item` enum (with the other ck3 items), and once in the `path` function.

File: src/ck3/tables/effects.rs [hunk 3]
```diff
@@ -507,7 +507,7 @@ const SCOPE_EFFECT: &[(Scopes, &str, Effect)] = &[
     (Scopes::Character, "refund_perks", Item(Item::Lifestyle)),
     (Scopes::Character, "release_from_prison", Yes),
     (Scopes::Accolade, "remove_acclaimed_knight", Yes),
-    (Scopes::Scheme, "remove_agent_slot", Item(Item::AgentType)),
+    (Scopes::Scheme, "remove_agent_slot", Scope(Item::AgentType)),
```
Comment (amtep) [inline src/ck3/tables/effects.rs:510]: It should now be `Scopes::AgentSlot` instead of the item type :)

**Review by amtep — CHANGES_REQUESTED** [review]
Some comments and questions

File: src/ck3/tables/effects.rs (hunk 3 above)
Comment (ProZeratul) [reply src/ck3/tables/effects.rs]: (Scopes::Scheme, "remove_agent_slot", Scope(Scopes::AgentType)),

File: src/ck3/data/messages.rs (hunk 1 above)
Comment (ProZeratul) [reply src/ck3/data/messages.rs]: Removed from file

File: src/ck3/data/messages.rs (hunk 2 above)
Comment (ProZeratul) [reply src/ck3/data/messages.rs]: Removed from file

**Review by amtep — APPROVED** [review]
Looks good. I'll run the cargo fmt myself.

**Pull request merged**